### DISCUSSION TITLE
Lazer's color hit meter lacks distinct small and large droplet colors

### DISCUSSION THREAD

**Discussion by nagi-desuuu** [ppy/osu#20927] [Q&A] [answered]
As per title says. The videos attached should probably explain things, although I probably should've searched for this first before opening...

Stable:

https://user-images.githubusercontent.com/75596775/197782581-b0bb0466-b738-463d-83e1-54b013efa101.mp4

Lazer: 

https://user-images.githubusercontent.com/75596775/197782808-4d832dbf-3b17-4d02-b9ff-0a12e2dde571.mp4

**Comment by bdach** [comment, marked as answer]
This only "worked" in stable because large droplets internally were "100s" and small droplets were "50s". Now they're separate object types that all get judged as "perfect", so the meter can't work like that.

I sort of question the usefulness of this meter in catch. Does anyone use this coloring "feature" for anything?
